contrib/python/pypodman/pypodman/lib/actions/_create_args.py

```diff
@@ -0,0 +1,394 @@
+            help='Publish all exposed ports to random'
+            ' ports on the host interfaces'
+            '(Default: False)')
+        parser.add_argument(
+            '--quiet',
+            '-q',
+            action='store_true',
+            help='Suppress output information when pulling images')
+        parser.add_argument(
+            '--read-only',
+            action=BooleanAction,
+            help="Mount the container's root filesystem as read only.")
+        parser.add_argument(
+            '--rm',
+            action=BooleanAction,
+            default=False,
+            help='Automatically remove the container when it exits.')
+        parser.add_argument(
+            '--rootfs',
+            action='store_true',
+            help=('If specified, the first argument refers to an'
+                  ' exploded container on the file system of remote host.'))
+        parser.add_argument(
+            '--security-opt',
+            action='append',
+            metavar='OPT',
+            help='Set security options.')
+        parser.add_argument(
+            '--shm-size', action=UnitAction, help='Size of /dev/shm')
+        parser.add_argument(
+            '--sig-proxy',
+            action=BooleanAction,
+            default=True,
+            help='Proxy signals sent to the podman run'
+            ' command to the container process')
+        parser.add_argument(
+            '--stop-signal',
+            metavar='SIGTERM',
+            help='Signal to stop a container')
+        parser.add_argument(
+            '--stop-timeout',
+            metavar='TIMEOUT',
+            type=int,
+            default=10,
+            help='Seconds to wait on stopping container.')
+        parser.add_argument(
+            '--subgidname',
+            metavar='MAP',
+            help='Name for GID map from the /etc/subgid file')
+        parser.add_argument(
+            '--subuidname',
+            metavar='MAP',
+            help='Name for UID map from the /etc/subuid file')
+        parser.add_argument(
+            '--sysctl',
+            action='append',
+            help='Configure namespaced kernel parameters at runtime')
+        parser.add_argument(
+            '--tmpfs', metavar='MOUNT', help='Create a tmpfs mount')
+        parser.add_argument(
+            '--tty',
+            '-t',
+            action=BooleanAction,
+            default=False,
+            help='Allocate a pseudo-TTY for standard input of container.')
+        parser.add_argument(
+            '--uidmap', metavar='MAP', help='UID map for the user namespace')
+        parser.add_argument('--ulimit', metavar='OPT', help='Ulimit options')
+        parser.add_argument(
+            '--user',
+            '-u',
+            help=('Sets the username or UID used and optionally'
+                  ' the groupname or GID for the specified command.'))
+        parser.add_argument(
+            '--userns',
+            choices=['host', 'ns'],
```

Comment:
Do we support container?  
--userns container:CONTAINERID?

I think choices might be incorrect here
--userns ns:PATHTOUSERNS?

Reply:
I changed this to pass value to service for validation.